src/context/ConversationContext.tsx

```diff
@@ -0,0 +1,24 @@
+import {createContext} from 'react';
```

Comment:
Setting up some sort of usage for Replies/Reactions, but realized the PR was getting big so for now functionality is a placeholder